web/src/components/AddAuthor/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import TextField from 'material-ui/TextField'
+import RaisedButton from 'material-ui/RaisedButton'
+import * as api from '../../services/API'
+const style = {
+    margin: "0 25%",
+  };
+
+ 
+export default class AddAuthor extends React.Component{
+    constructor(props){
+        super(props);
+        this.state={
+            name: ''
+        };
+
+        this.handleChange = this.handleChange.bind(this);
+        this.handleAddClick = this.handleAddClick.bind(this);
+
+    }
+
+    handleChange = (event) =>{
+        this.setState({name: event.target.value});
+    }
+
```

Comment:
How I can hide form and show for example dialog "succes"?

Reply:
I see 2 options:
1) Redirect `form save` to another route. Like you have `/books` and `/books/create` in `<Routes>...</Routes>`. `/books` - show all books. `/books/create` - Create single book. When you click `Create book`, it shows only create book form. When you click `Save`, it redirects to `/books` and show notification `Success` that disappears in 3 seconds.
2) Do everything on one route and do smth like `this.setState({showNotification: true})` when book is saved